### PR TITLE
bareUcj config setup + easydao log update 

### PR DESCRIPTION
**Pull request by jinnnn9**
address:
https://nanopay.atlassian.net/browse/NP-5813
https://nanopay.atlassian.net/browse/NP-5814

- https://github.com/foam-framework/foam2/pull/4878 - pr that disabled interface decorators on bareUcjDAO

### CODE REVIEW

**Review by jlhughes — APPROVED** [review]

File: src/foam/nanos/crunch/services.jrl [hunk 1]
```diff
@@ -79,6 +79,7 @@ p({
       .setAuthorize(false)
       .setInnerDAO(x.get("bareUserCapabilityJunctionDAO"))
       .setOf(foam.nanos.crunch.UserCapabilityJunction.getOwnClassInfo())
+      .setEnableInterfaceDecorators(false)
```
Comment (Nauna) [inline src/foam/nanos/crunch/services.jrl:82]: So a UCJ has a few interfaces it uses
CreatedAware
LifecycleAware
...
can find them in UserCapabilityJunctionRefine.js

so if we want to turn anything off it should be just one of those... shouldn't be all with the `.setEnableInterfaceDecorators(false)`

is it possible to just remove this and run some tests. Its not clear at all why this is here (bareUser or ucj )

Comment (jinnnn9) [reply src/foam/nanos/crunch/services.jrl:82]: The interface decorators were both on ucjDao and bareUcjDao, so they were triggered twice, and that is the reason for disabling on the higher level of the decorator chain.

Comment (jinnnn9) [reply src/foam/nanos/crunch/services.jrl:82]: **decorator chain without disable**
ProxyDAO
EasyDAO
ProxyDAO
_LastModifiedByAwareDAO
LastModifiedAwareDAO
CreatedByAwareDAO
CreatedAwareDAO_
RulerDAO
_LifecycleAwareDAO_
UserCapabilityJunctionDAO
ProxyDAO
EasyDAO
ProxyDAO
_LastModifiedByAwareDAO
LastModifiedAwareDAO
CreatedByAwareDAO
CreatedAwareDAO_
RulerDAO
_LifecycleAwareDAO_
HistoryDAO
GUIDDAO
JDAO
MDAO

Comment (jinnnn9) [reply src/foam/nanos/crunch/services.jrl:82]: **decorator chain with disable**
ProxyDAO
EasyDAO
ProxyDAO
RulerDAO
UserCapabilityJunctionDAO
ProxyDAO
EasyDAO
ProxyDAO
LastModifiedByAwareDAO
LastModifiedAwareDAO
CreatedByAwareDAO
CreatedAwareDAO
RulerDAO
LifecycleAwareDAO
HistoryDAO
GUIDDAO
JDAO
MDAO

Comment (Nauna) [reply src/foam/nanos/crunch/services.jrl:82]: interesting!

**Review by Nauna — APPROVED** [review]

**Pull request merged**